XCUITests/TopTabsTest.swift

```diff
@@ -160,4 +160,26 @@ class TopTabsTest: BaseTestCase {
         waitforNoExistence(app.collectionViews.cells[urlLabel])
         checkNumberOfTabsExpectedToBeOpen(expectedNumberOfTabsOpen: 1)
     }
+
+    func testCloseTabFromPageOptionsMenu() {
+        navigator.openURL(url)
+        XCTAssertTrue((navigator.userState.url?.starts(with: "www.mozilla.org"))!)
+        navigator.goto(PageOptionsMenu)
+        navigator.goto(HomePanelsScreen)
+        XCTAssertEqual(navigator.screenState, HomePanelsScreen)
+
+        navigator.goto(TabTray)
+        checkNumberOfTabsExpectedToBeOpen(expectedNumberOfTabsOpen: 1)
+    }
```

Comment:
The test should

a) open two new tabs
b) check that the number of tabs open is 2.
c) close a tab
d) check that the number of tabs is 1.
c) close a tab
d) check that the number of tabs is 1.

I'm not sure you need to open any URLs, just perform the `Action.NewTab` a couple of times, and `Action.CloseTabFromPageOptionsMenu`.

You should also put `checkNumberOfTabsExpectedToBeOpen` into its own method. Either on `navigator` or the test.

```swift
func checkNumberOfTabsExpectedToBeOpen(_ numTabs) {
    navigator.goto(TabTray)
    XCTAssertEqual(userState.numTabs, numTabs)
}

Reply:
+1, we probably don't need to access external URL for this test, as we can create multiple empty tabs.

Reply:
We need to open an URL otherwise the Page Options menu is not accesible. I can open a few tabs without going to an URL, but one has to be a website...